src/views/Vote/components/ProposalTable.tsx

```diff
@@ -43,46 +49,63 @@ const ProposalTable: React.FC = () => {
             currentTime <= indexProposals[id]?.msg?.payload?.end
           const isExecuted = indexProposals[id]?.msg?.payload?.end < currentTime
 
-          return (
-            <>
-              <StyledColumnRow>
-                <StyledColumn style={{ width: '60%' }}>
-                  {indexProposals[id].msg.payload.name}
-                </StyledColumn>
-                <StyledColumn style={{ width: '20%' }} isActive={isActive}>
-                  {isActive && 'Active'}
-                  {isExecuted && 'Executed'}
-                </StyledColumn>
-                <StyledColumn style={{ width: '20%' }}>
-                  <StyledButton
-                    onClick={() => {
-                      window.open(
-                        `${SNAPSHOT_INDEX_PROPOSAL_BASE}/${id}`,
-                        '_blank'
-                      )
-                    }}
-                    isActive={isActive}
-                  >
-                    View
-                  </StyledButton>
-                </StyledColumn>
-              </StyledColumnRow>
-              {i !== proposalIds.length - 1 ? (
-                <StyledDivider />
-              ) : (
-                <Spacer size='md' />
-              )}
-            </>
-          )
+          if (
+            !searchTerm ||
+            indexProposals[id].msg.payload.name
```

Comment:
This seems prone to breakage. Maybe add the `?` operator to protect against one of these fields not existing.

Reply:
Also assigning this if statement to a variable (like isActive or isExecuted above) will make it more readable:
`const shouldDisplayProposal = !searchTerm || indexProposals[id]....`

Reply:
Got it, will fix tonight. Thanks for review!

Reply:
Not sure I 100% understand you on the breakage, could you expand on that?